### PR TITLE
feat: op-deployer upgrade, fix fork support

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

File: network_params.yaml [hunk 1]
```diff
@@ -34,17 +34,15 @@ optimism_package:
         seconds_per_slot: 2
         name: "op-kurtosis"
         fjord_time_offset: 0
-        granite_time_offset: ""
-        holocene_time_offset: ""
-        interop_time_offset: ""
+        granite_time_offset: 0
+        holocene_time_offset: 0
```
[the remark this answers is not in the document]
Comment (mslipper) [reply network_params.yaml:38]: Granite should be activated, holocene should not.

File: src/package_io/input_parser.star [hunk 2]
```diff
@@ -252,8 +259,9 @@ def default_participant():
 
 def default_op_contract_deployer_params():
     return {
-        "image": "mslipper/op-deployer:latest",
-        "artifacts_url": "https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-4accd01f0c35c26f24d2aa71aba898dd7e5085a2ce5daadc8a84b10caf113409.tar.gz",
+        "image": "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-deployer:v0.0.6",
+        "l1_artifacts_locator": "tag://op-contracts/v1.6.0",
```
[the remark this answers is not in the document]
Comment (mslipper) [reply src/package_io/input_parser.star:263]: It needs to be tag:// - that's what tells op-deployer to use a fixed URL for these artifacts. I'll fix the other one.

File: README.md [hunk 3]
```diff
@@ -219,7 +219,7 @@ optimism_package:
         fjord_time_offset: 0
 
         # Granite fork
-        # Defaults to None - not activated - decimal value
+        # Defaults to 0 (genesis activation) - decimal value
         # Offset is in seconds
         granite_time_offset: ""
```
Comment (barnabasbusa) [inline README.md:224]: if it defaults to 0 then maybe set granite_time_offset: 0 :D

File: src/package_io/input_parser.star (hunk 2 above)
[the remark this answers is not in the document]
Comment (barnabasbusa) [reply src/package_io/input_parser.star:263]: pls fix the readme too then.

File: README.md (hunk 3 above)
Comment (mslipper) [reply README.md]: my bad, nice catch

**Pull request merged**